examples/13. Authentication.md

```diff
@@ -0,0 +1,57 @@
+FORMAT: 1A
+
+# Authentication API
+
+## API Blueprint
++ [Previous: Advanced Action](12.%20Advanced%20Action.md)
++ [This: Raw API Blueprint](https://raw.github.com/apiaryio/api-blueprint/master/examples/13.%20Authentication.md)
+
+## Authentication
+The allowed authentication schemes for the API server are defined below.
+
++ Authentication Schemes
+    + basic (Basic Auth Scheme) - Using the username and password directly
+    + oauth (Oauth2 Auth Scheme)
+        + authorization_endpoint: `/authorize`
+        + access_token_endpoint: `/access_token`
+        + token_header_keyword: `token`
+        + token_query_keyword: `access_token`
+
++ Response 403 (application/json)
+
+        {
+          "error": "Requires Authentication"
+        }
+
+## User [/user]
+
+### Retrieve a User Profile [GET]
+This endpoint needs authentication from the user before retrieving his profile data.
+
++ Authentication
+
++ Response 200 (application/json)
+
+        {
+          "username": "bond",
```

Comment:
Check what payloads are used throughout other examples. I think it is worth to try to keep this unified. That way it will be less confusing for those user that are browsing through examples as whole.